.eslintrc.js

```diff
@@ -26,7 +26,10 @@ module.exports = {
         "import/no-extraneous-dependencies": ["error", {"devDependencies": true}]
       },
       "settings": {
-        "import/core-modules": Object.keys(devDependencies)
+        "import/core-modules": [
+          'govuk-react',
```

Comment:
curious as to what this change does :grin:

Reply:
https://github.com/UKHomeOffice/govuk-react/pull/451/files#diff-77150f3501b5f8d820e6f61a753cb5f9R2

I'm importing from govuk-react in a fixture rather than importing from the specific sub packages. Because it's easier and feels better.

This was coming up with an eslint error as govuk-react wasn't included in the devDependencies. I'm adding it as an exception in our eslint rules as with all the other monorepo packages.

Reply:
ah ok - cool :grin: